test/unit/specs/components/ActionModal/components/__snapshots__/ActionModal.spec.js.snap

```diff
@@ -98,6 +98,7 @@ exports[`ActionModal should show the action modal when user has logged in with e
      
     <tableinvoice-stub
       amount="0"
+      bonddenom="uatom"
```

Comment:
why is this `uatom` if `bondDenom: `STAKE``

Reply:
Would this not be determined by the test? https://github.com/luniehq/lunie/blob/f929ee5d1f8d831588fdd7fb5287bbbb638077e5/src/ActionModal/components/ActionModal.vue#L62

Reply:
yep yep. my mistake.